src/parsers/forecast_parser.jl

```diff
@@ -1,16 +1,60 @@
+
+const CATEGORY_STR_TO_COMPONENT = Dict{String, DataType}(
+    "Bus" => Bus,
+    "Generator" => Generator,
+    "Reserve" => Service,
+    "LoadZone" => LoadZones,
+    "ElectricLoad" => ElectricLoad,
```

Comment:
Why is the referencing made to Abstract Types? I thought we had a default concrete struct to refer to.